imageroot/events/user-domain-changed/20configure_ldap

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2024 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import sys
+import os
+import agent
+
+rdb = agent.redis_connect(use_replica=True)
+event = json.load(sys.stdin)
+
+if not os.environ.get('MAIL_SERVER', ''):
+    exit(0) # ignore event if mail server is not set
+
+user_domain = ''
+
+
+if not os.environ.get('USER_DOMAIN'):
+    providers = agent.list_service_providers(rdb, 'imap', 'tcp', {
+        'module_uuid': os.environ.get('MAIL_SERVER', '')
+    })
+
+    if providers:
+        user_domain = providers[0]['user_domain']
+
+if event.get('domain') != os.getenv('USER_DOMAIN', user_domain):
+    exit(0)
+
+if 'node_id' in event and str(event['node_id']) != os.getenv('NODE_ID'):
+    exit(0) # ignore event if the source is not in our node
```

Comment:
The account provider can be on another node, so ignore the info: we must always restart regardless of the node_id inside the event.
```suggestion
```